patches/react-native-gesture-handler+2.20.0.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/node_modules/react-native-gesture-handler/android/fabric/src/main/java/com/swmansion/gesturehandler/ReactContextExtensions.kt b/node_modules/react-native-gesture-handler/android/fabric/src/main/java/com/swmansion/gesturehandler/ReactContextExtensions.kt
+index ff440de..2e05de9 100644
+--- a/node_modules/react-native-gesture-handler/android/fabric/src/main/java/com/swmansion/gesturehandler/ReactContextExtensions.kt
++++ b/node_modules/react-native-gesture-handler/android/fabric/src/main/java/com/swmansion/gesturehandler/ReactContextExtensions.kt
+@@ -8,5 +8,5 @@ import com.facebook.react.uimanager.events.Event
+ 
+ fun ReactContext.dispatchEvent(event: Event<*>) {
+   val fabricUIManager = UIManagerHelper.getUIManager(this, UIManagerType.FABRIC) as FabricUIManager
+-  fabricUIManager.getEventDispatcher().dispatchEvent(event)
++  fabricUIManager.eventDispatcher.dispatchEvent(event)
```

Comment:
Did you raise that with the gesture handler team? If not, I can do that.

Reply:
Haven't raised it yet, so feel free to contact them